content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+
+1. `--increment-instance-enabled=true` states that we want to be able to execute this Composed Task multiple times. (Composed Tasks are build using Spring Batch and thus are batch jobs)
+1. `--interval-time-between-checks=1000` states that the `Composed Task Runner` will wait 1 second between checks to make sure that a task is complete (the default is 10 seconds).
+
+We used the properties to set the transition-sample properties. Notice we had 4 sections for the taskapp.exitMessage property lets breakdown the `app.basictransition.transition-sample.taskapp.exitMessage=FAILED` into its components:
+
+1. `app` - tells spring cloud dataflow that this property is to be used for task application.
+1. `basictransition` - tells dataflow that this property will be used by the `Composed Task Runner` or one of the apps in the graph
+1. `transition-sample` - tells dataflow that this property is to be used by the transition-sample
+1. `taskapp.exitMessage` - the property itself.
+
+More can be read about the sections of a property and the different property types [here](#passing-properties)
+
+#### Are there more states to a transition?
+
+Now what happens if I were to enter "FOO" for the exit message what would happen? Well lets try it!
+
+To do this let's select the `basictransition` `Composed Task Runner` to be executed as shown below:
+![Transition Execution Flow Launch](images/SCDF-composed-task-transition-launch.png)
+Now from the task launch page lets populate page with the following:
+Arguments:
+
+```
+--increment-instance-enabled=true
+--interval-time-between-checks=1000
+```
+
+Parameters:
+
+```
+app.basictransition.transition-sample.taskapp.exitMessage=FOO
+```
+
+It should look like the following:
+
+![Transition Execution Flow Launch-Config-FOO](images/SCDF-composed-task-transition-launch-foo-fail.png)
+
+Now that it has been executed let’s verify that path `FOO` was actually followed and this can be done by pressing the `Executions` tab at the top of the task page:
+![Transition Execution Flow Launch-FOO-LIST](images/SCDF-composed-task-transition-launch-foo-fail-list.png)
+
+In this case we see that the composed task ended with just running the `Composed Task Runner` and the transition sample. This was because FOO was not targeted. How would we handle that? i.e. have a path for COMPLETED, FAILED, and everything else?
+
+In this case would want to create another composed task using a wild card that would look like:
+
+To create your basic transition using the Spring Cloud Data Flow UI, press the `Tasks` tab on the left hand side of the dashboard and then press the `Create Task(s)` button at the top of the page.
+Now copy the expression below and paste it in the text box located at the top of the page:
+
+```
+transition-sample 'FAILED'->timestamp-1: timestamp 'COMPLETED'->timestamp-2: timestamp '*' -> timestamp-3:timestamp
```

Comment:
I find it hard to read this.  also, maybe there shouldn't be a space after the colon - also explain the use of labels as that makes it look much more complex.

Reply:
Can we maybe register timestamp multiple times so that we avoid having to use labels?